Tabloid/client/src/components/TagCreate.js

```diff
@@ -0,0 +1,50 @@
+import React, { useEffect, useState  } from 'react';
+import { Form } from 'reactstrap';
+import { addTag } from '../modules/TagManagment';
+import Tags from './TagView';
+import { Link } from 'react-router-dom';
+import { useHistory } from 'react-router-dom';
```

Comment:
3 of these imported items are not used currently. Do you intend to utilize them in the future or is it okay to remove them from our import statements?

Reply:
@SchmidtFred removed the unused items from the TagCreate.js